src/lib/dnssd/Types.h

```diff
@@ -220,19 +221,22 @@ struct DnssdNodeData
     char deviceName[kMaxDeviceNameLen + 1]                    = {};
     char pairingInstruction[kMaxPairingInstructionLen + 1]    = {};
 
-    DnssdNodeData() {}
+    CommissionNodeData() {}
 
     void Reset()
     {
+        CommonResolutionData::Reset();
         // Let constructor clear things as default
-        this->~DnssdNodeData();
-        new (this) DnssdNodeData();
+        this->~CommissionNodeData();
+        new (this) CommissionNodeData();
     }
 
     bool IsInstanceName(const char * instance) const { return strcmp(instance, instanceName) == 0; }
 
     void LogDetail() const
     {
+        CommonResolutionData::LogDetail();
```

Comment:
This is losing the "Discovered node:" log we used to have, right? Is there a way we can keep that?

For example, maybe `DiscoveredNodeData` should inherit from the variant instead of being the variant.  Then it could implement a LogDetail() that logs "Discovered node:" and then calls LogDetail() on the appropriate branch of the variant....

Reply:
Added a log _"Discovered commissionable/commissioners node:"_ before logging `CommonResolutionNode`